browser/ui/webui/skus_internals_ui.cc

```diff
@@ -36,4 +41,15 @@ void SkusInternalsUI::GetEventLog(GetEventLogCallback callback) {
   NOTIMPLEMENTED_LOG_ONCE();
 }
 
+void SkusInternalsUI::GetSkusState(GetSkusStateCallback callback) {
+  // TODO(simonhong): Determine which value should be displayed.
```

Comment:
@bsclifton needs input here

Reply:
This should be `kSkusState` in local state as a dictionary (key name is `skus.state`)

But then there are going to be several entries that look like this:
```json
{
    "skus": {
        "state": {
            "migrated_to_local_state": true,
            "skus:development": "raw-state-here (JSON with quotes escaped)",
            "skus:production": "raw-state-here (JSON with quotes escaped)",
            "skus:staging": "raw-state-here (JSON with quotes escaped)"
        }
    }
}
```

Reply:
Should be able to add some methods to https://github.com/brave/brave-core/blob/master/components/skus/browser/skus_utils.cc